File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/Location.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Location {

    @SerializedName("position")
    @Expose
    private List<String> position = null;

    public List<String> getPosition() {
        return position;
    }

    public void setPosition(List<String> position) {
        this.position = position;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverHere/Location.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverHere

import com.google.gson.annotations.Expose

class Location {
    @SerializedName("position")
    @Expose
    private var position: List<String>? = null
    fun getPosition(): List<String>? {
        return position
    }

    fun setPosition(position: List<String>?) {
        this.position = position
    }
}